### PR TITLE
New Instant Answer: Apple Docs tvOS Fathead

### PR DESCRIPTION
**Pull request by souravbadami**
https://duck.co/ia/view/apple_docs_tvos

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@souravbadami I've updated the ID to **apple_docs_tvos** (following suite with https://duck.co/ia/view/apple_docs_ios and https://duck.co/ia/view/apple_docs_macos).

Can you please change the directory here to `apple_docs_tvos` to match the ID?

File: lib/fathead/tvos/fetch.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+mkdir -p download
+cd download
+rm -f *.tgz
+wget http://kapeli.com/feeds/tvOS.tgz
+tar -vxjf tvOS.tgz
```
Comment (moollaza) [inline lib/fathead/tvos/fetch.sh]: typo? `-xvzf`

Comment (souravbadami) [inline lib/fathead/tvos/fetch.sh]: Yep, Solved.

**Comment by souravbadami** [comment]
@moollaza -- It's solved now. PTAL.

**Comment by moollaza** [comment]
@souravbadami perfect, thanks.

This is on Beta now. Just need to re-deploy it to fix the More At link...

**Comment by moollaza** [comment]
https://beta.duckduckgo.com/?q=TVApplicationControllerContext&t=hk&ia=about

**Comment by moollaza** [comment]
@souravbadami Thanks! This LGTM :+1:

**Comment by daxtheduck** [comment]
@souravbadami this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**